server/pulp/server/controllers/repository.py

```diff
@@ -454,14 +454,17 @@ def queue_delete(repo_id):
 
 def get_importer_by_id(object_id):
     """
-    Get a plugin and call configuration using the document ID
+    Get a plugin, call configuration, and Importer document object using the document ID
```

Comment:
I knew you would like that @jeremycline.

http://www.theonion.com/article/4-copy-editors-killed-in-ongoing-ap-style-chicago--30806